src/main/java/org/broadinstitute/hellbender/utils/dataflow/DataflowUtils.java

```diff
@@ -67,6 +73,50 @@ public void processElement(ProcessContext c) {
                 .apply(ParDo.of(new LoadReadsFromFileFn(intervals)));
     }
 
+    /**
+     * Ingest BAM files from a Hadoop file system and load them into a
+     * <code>PCollection<Read></code>.
+     * @param pipeline a configured Pipeline
+     * @param intervals intervals to select reads from
+     * @param path Hadoop file or directory path to read BAM files from
+     * @return a <code>PCollection<Read></code> with all the reads that overlap the
+     * given intervals in the BAM files
+     */
+    public static PCollection<Read> getReadsFromHadoopBams(final Pipeline pipeline, final List<SimpleInterval> intervals, final String path) {
```

Comment:
Does this actually read multiple bam files if you provide directory with multiple files as the path?  

If that is the case could you change it so that it accepts a list of paths instead and only reads from the explicitly specified bams?  

If it only reads from 1 at a time, could you change the name/ documentation.